e2e/testcases/death/helpers.ts

```diff
@@ -0,0 +1,240 @@
+import gql from 'graphql-tag'
+import { print } from 'graphql/language/printer'
+import { GATEWAY_HOST } from '../../constants'
+import { DeathRegistrationInput } from '../../gateway'
+import faker from '@faker-js/faker'
+
+import { readFileSync } from 'fs'
+import uuid from 'uuid'
+import { format, subDays, subYears } from 'date-fns'
+import { Bundle } from 'typescript'
+import { join } from 'path'
+import { declaration } from './data/declarationData'
+
+export const CREATE_DEATH_REGISTRATION = print(gql`
+  mutation createDeathRegistration($details: DeathRegistrationInput!) {
+    createDeathRegistration(details: $details) {
+      trackingId
+      compositionId
+      isPotentiallyDuplicate
+      __typename
+    }
+  }
+`)
+
+async function getAllLocations(
+  type: 'ADMIN_STRUCTURE' | 'HEALTH_FACILITY' | 'CRVS_OFFICE'
+) {
+  const locations = (await fetch(
+    `${GATEWAY_HOST}/location?type=${type}&_count=0`
+  ).then((res) => res.json())) as fhir.Bundle
+  return locations.entry!.map((entry) => entry.resource as fhir.Location)
+}
+
+function getLocationIdByName(locations: fhir.Location[], name: string) {
+  const location = locations.find((location) => location.name === name)
+  if (!location) {
```

Comment:
```suggestion
  if (!location?.id) {
```
This should make the return type of this function easier to deal with